### PR TITLE
Automation - Initial configuration

### PR DESCRIPTION
**Pull request by bubafinder**
Introducing Cypress to Tailor:
- initial configuration
- two basic tests added (running on port 3000)

### CODE REVIEW

File: cypress/fixtures/credentials.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (underscope) [inline cypress/fixtures/credentials.json:1]: We can pick it up from seed to make it less prone to future changes.

File: cypress/framework/selectors/login.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const alertContent = '//div[@class="v-alert__content"]';
```
Comment (underscope) [inline cypress/framework/selectors/login.js:3]: https://docs.cypress.io/guides/references/best-practices
@MiroDojkic @kronicker @vladimyr  Should we conform to => https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Comment (vladimyr) [reply cypress/framework/selectors/login.js:3]: Mr React gave us the answers we need, no need for second thoughts 😄 
https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change#whats-with-the-data-testid-query

Comment (underscope) [reply cypress/framework/selectors/login.js:3]: 'Mr React' is using testing-library https://testing-library.com/docs/queries/bytestid.

`
Using data-testid attributes do not resemble how your software is used and should be avoided if possible. That said, they are way better than querying based on DOM structure or styling css class names.
`

Everything makes purrfect sense now 🤡

**Comment by underscope** [comment]
@markolovric Closing this one in favor of avoiding Page Objects and xpath 😉

**Pull request closed without merging**